pkg/scheduler/factory/factory.go

```diff
@@ -308,19 +309,26 @@ func NewConfigFactory(
 		podQueue:   c.podQueue,
 	}
 
-	ch := make(chan os.Signal, 1)
-	signal.Notify(ch, compareSignal)
+	if resyncIntervalInSec != 0 {
```

Comment:
I think it would be useful to keep the existing logic where we can send a signal to scheduler to trigger a cache compare even when `--resync-interval` is not provided.

Reply:
Yes, I also like the signal trigger

Reply:
ok. Will do.